lib/ddtrace/contrib/grape/configuration/settings.rb

```diff
@@ -24,6 +24,12 @@ class Settings < Contrib::Configuration::Settings
           end
 
           option :service_name, default: Ext::SERVICE_NAME
+          option :error_responses, default: '500-599' do |o|
+            o.default { '500-599' }
```

Comment:
I believe the default should be `Datadog::Contrib::StatusCodeMatcher.new(`500-599`)`, if i'm understanding things correctly.

also let's abstract out `500-599` into a constant within the `ddtrace/ext/http` module, that will help with keeping things DRY

Reply:
Hmm wondering if it makes sense to just use the `ERROR_RANGE` that's already provided. With that, I can do something like this:
```ruby
option :error_responses, default: '500-599' do |o|
    o.default { (Datadog::Ext::HTTP::ERROR_RANGE).to_a }
    ...
```

This way, we get functionality from an existing variable rather than defining a new and similar constant

Reply:
sure i think that works just fine